Source/VCGeneration/ConditionGeneration.cs

```diff
@@ -635,8 +635,6 @@ public enum Outcome {
     protected string/*?*/ logFilePath;
     protected bool appendLogFile;
 
-    public static List<Model> errorModelList;
```

Comment:
This field appears to be redundant because the model  is available as a field of Counterexample itself.  Hence I removed it and simplified the code in a particular variation of VerifyImplementation (see below).